src/VAC/VectorAnimationComplex/VAC.h

```diff
@@ -483,6 +489,8 @@ public slots:
 
     double pasteDeltaX_;
     double pasteDeltaY_;
+
+    bool isManualTransform;
```

Comment:
add _ suffix to match other member var names?

Reply:
Done